File: Secuenciales/02.py
import os
os.system("cls")

metros = int(input("metros : "))

centimetros = metros * 100.0
Pulgadas = centimetros / 2.54
Pies = Pulgadas / 12
Yardas = Pies / 3

print(f"Centimetros = {centimetros:.2f}")
print(f"Pulgadas = {Pulgadas:.2f}")
print(f"Pies = {Pies:.2f}")
print(f"Yardas = {Yardas:.2f}")

File: JAVA/Secuenciales/_02.java
// Desarrolle  el  programa  que  permite  convertir  una  cantidad  dada  en  metros  a  su  equivalente  en centímetros, pulgadas, pies y yardas. Considere los siguientes factores de conversión:   1 metro = 100 cm, 1 pie = 12 pulgadas, 1 yarda = 3 pies, 1 pulgada = 2.54 cm

package JAVA.Secuenciales;

import java.util.Scanner;

public class _02 {
    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Ingrese la cantidad de metros : ");
        int metros = scanner.nextInt();

        double centimetros = metros * 100;
        double yardas = metros *1.093613;
        double pies = metros * 3.28084;
        double pulgadas = metros * 39.37008;

        System.out.println("Transformado a cm es igual a : " + centimetros + " cm");
        System.out.println("Transformado a yd es igual a : " + yardas + " yd");
        System.out.println("Transformado a Pe es igual a : " + pies + " Pies");
        System.out.println("Transformado a Pg es igual a : " + pulgadas + " pg");

        scanner.close();
    }
}
